datafusion/proto/src/logical_plan/to_proto.rs

```diff
@@ -489,50 +489,59 @@ impl TryFrom<&Expr> for protobuf::LogicalExprNode {
                     // linearized from left innermost to right outermost (but while
                     // traversing the chain we do the exact opposite).
                     operands: exprs
-                    .into_iter()
-                    .rev()
-                    .map(|expr| expr.try_into())
-                    .collect::<Result<Vec<_>, Error>>()?,
+                        .into_iter()
+                        .rev()
+                        .map(|expr| expr.try_into())
+                        .collect::<Result<Vec<_>, Error>>()?,
                     op: format!("{op:?}"),
                 };
                 Self {
                     expr_type: Some(ExprType::BinaryExpr(binary_expr)),
                 }
             }
-            Expr::Like(Like { negated, expr, pattern, escape_char }) => {
+            Expr::Like(Like {
+                negated,
+                expr,
+                pattern,
+                escape_char,
+            }) => {
                 let pb = Box::new(protobuf::LikeNode {
                     negated: *negated,
                     expr: Some(Box::new(expr.as_ref().try_into()?)),
                     pattern: Some(Box::new(pattern.as_ref().try_into()?)),
-                    escape_char: escape_char
-                        .map(|ch| ch.to_string())
-                        .unwrap_or_default()
+                    escape_char: escape_char.map(|ch| ch.to_string()).unwrap_or_default(),
                 });
                 Self {
                     expr_type: Some(ExprType::Like(pb)),
                 }
             }
-            Expr::ILike(Like { negated, expr, pattern, escape_char }) => {
+            Expr::ILike(Like {
```

Comment:
these changes look fine to me, but it isn't clear to me why `rustfmt` / lint didn't catch them before 🤔